Documentation/Plus Codes Specification.md

```diff
@@ -0,0 +1,119 @@
+# Plus Codes Specification
+
+## Version history
+
+All substantial changes changes to this document are listed here.
+
+* Version 1.1.0 / 2021-06-04 / [William Entriken](https://github.com/fulldecent/)
+  * Separate Plus Codes specification from [Open Location Code API specification](./Open Location Code API Specification.md)
+  * Define codes for north and south poles
+  * Specify which code levels of full codes may be shortened to which code levels of short codes
+  * Establish consistent wording/naming
+* Version 1.0.0 / 2019-04-29 / [Doug Rinckes](https://github.com/drinckes), Google / Philipp Bunge, Google
+  * Initial public release
+
+## Area and bounds
+
+A **Plus Code** represents a locus of coordinates ("**area**") with **bounds** on northern & southern parallels and western & eastern meridians. This area includes exactly the coordinates:
+
+1. Inside (excluding) the bounds;
+2. On the western bound between (excluding) the northern and southern bounds;
+3. (If the northern bound is 90° N and the western bound is 0° W) the North Pole (i.e. the point at 90° N);
+4. (If the southern bound is 90° S and the western bound is 0° W) the South Pole (i.e. the point at 90° S); and
+5. (If the southern bound is not 90° S) on the southern bound from (including) the western bound to (excluding) the eastern bound.
+
+This specification references latitudes and longitudes on Earth under [WGS 84](https://earth-info.nga.mil), which is the standard used by the Global Positioning System. If another geodetic is used, that context MUST be agreed between the producer and consumer of a Plus Code.
```

Comment:
Is it really the case / important that coordinates are generally referenced according to WGS84, or is it rather the case that producer and consumer need to agree on that? Suggestion: remove this here, but merge into a section on "necessary producer/consumer agreements" that also includes details about how shortening codes only works if producder and consumer can agree on reference locations.

Reply:
I think it is well understood that Plus Codes refer to places on Earth, offset based on the high noon sun location of Greenwich, England. Or in plain English, "these are GPS coordinates, and compatible with all your normal GPS stuff". Otherwise we are just talking about coordinates on a sphere. Happy to remove if that is already obvious to everyone, (but then should update Longest edge [formerly Precision] to reference Earth).

Currently the relevant sentence for short code Plus Codes is the first sentence in that section. 

> A **short code** Plus Code is meaningful only when the producer and consumer of the Plus Code can agree on the approximate latitude and longitude of a **reference location**.

Shall I add a note at the bottom of that section which details under which circumstances the producer and consumer will disagree an a shortening?

Reply:
I've actually already seen one application that uses plus codes for either Mars or the Moon. In the end, all that OLC does is to translate a 2-tuple of floating point numbers into a string and back again. The exact semantics of that process - whether it is current GPS coordinates on Earth, or coordinates in any other geodetic system on Earth, or coordinates on any other celestial body whether real or fictional - doesn't really matter outside of the necessary producer/consumer agreement.

The suggestion to specify "longest edge" to refer to Earth sounds good.

Reply:
Updated Earth note at 79ca2c2c94a42f7a44edeb0e52fc5e9524aac846

---

A shortening will work if the producer and consumer agree, to an approximate amount, of where a reference location is. Beyond some specific tolerance, the shortening/recovery will not be inverses.

You and I both understand this fuzziness but it is not spelled out. Shall I spell it out?